static/src/js/components/AccessMethod/AccessMethod.jsx

```diff
@@ -920,7 +930,9 @@ const AccessMethod = ({
           )
         }
         {
-          (supportsSwodlr && granuleList.length <= maxSwodlrGranuleCount) && (
+          (supportsSwodlr
+            && granuleList.length <= maxSwodlrGranuleCount
+            && !granuleListUndefined) && (
```

Comment:
This line is going to not have code coverage without a test testing the granuleList having an undefined value.

Reply:
There is a test there. That's why the coverage is 100% right now on code-cov 
![Uploading image.png…]()